content/applications/general/iot/config/troubleshooting.rst

```diff
@@ -343,6 +343,66 @@ Save`.
              :align: center
              :alt: Receipt format using naming convention: EpsonTMU220B__IMC_LDV_LDH_SCALE35__.
 
+DYMO LabelWriter 450 DUO print delay
+------------------------------------
+
+The DYMO LabelWriter 450 DUO has a known delay in printing with the :abbr:`IoT (Internet of Things)`
+box while using the label function. This is due to the printer being set up as a *Local RAW
+Printer*. To resolve this issue, open the OpenPrinting CUPS administration page from the :abbr:`IoT
+(Internet of Things)` box homepage and click :guilabel:`Add a Printer`.
+
+.. important::
+   The DYMO LabelWriter 450 DUO printer must already be connected and installed on the :abbr:`IoT
+   (Internet of Things)` box.
+
+.. image:: troubleshooting/add-printer-dymo.png
+   :align: center
+   :alt: Add a printer button highlighted on the Printer CUPS management page.
+
+From the :guilabel:`Local Printers` section select the :guilabel:`DYMO LabelWriter 450 DUO Label
+(DYMO LabelWriter 450 DUO Label)` pre-installed printer. Click :guilabel:`Continue`.
+
+.. image:: troubleshooting/local-printer.png
+   :align: center
+   :alt: Add a printer screen on OpenPrinting CUPS with DYMO LabelWriter 450 DUO Label highlighted.
+
+On the following screen, modify the :guilabel:`Name` to something recognizable as the original
+printer will still be present. Click :guilabel:`Continue` to be taken to the next screen.
+
+.. image:: troubleshooting/rename-printer.png
+   :align: center
+   :alt: Rename printer page in the 'Add a Printer' flow, with the name field highlighted.
+
+Next, choose the :guilabel:`Model`. Select :guilabel:`DYMO LabelWriter 450 DUO Label (en)`, and
+finally, click :guilabel:`Add Printer` to complete the installation.
+
+.. image:: troubleshooting/choose-printer.png
+   :align: center
+   :alt: Choose model screen on the OpenPrinting CUPS console with model and add a printer
+         highlighted.
+
+After being redirected to a confirmation page acknowledging a successful installation, click on the
+:guilabel:`Printers` button in the top menu.
+
+All the printers installed on the OpenPrinting CUPS server will appear, including the newly
+installed: *DYMO LabelWriter 450 DUO Label*. Click into the *DYMO LabelWriter 450 DUO Label* printer
```

Comment:
I will defer to Sam on this, but since you are directing the user to click on the specific printer, this might need something other than italics? (possibly backticks?_

Reply:
@samueljlieber 

Thoughts on this?

Reply:
@tiku-odoo 
If the text "DYMO LabelWriter 450 DUO Label" is visible on the **Printers** page, then it needs a :guilabel:
```
Click into the :guilabel:`DYMO LabelWriter 450 DUO Label` printer...
```
If the text is not visible you can use italics to refer to the object the user clicks that represents the printer.